src/data/finset/n_ary.lean

```diff
@@ -46,6 +46,11 @@ lemma card_image₂_le (f : α → β → γ) (s : finset α) (t : finset β) :
   (image₂ f s t).card ≤ s.card * t.card :=
 card_image_le.trans_eq $ card_product _ _
 
+lemma card_image₂_iff :
```

Comment:
```suggestion
lemma card_image₂_eq_iff_inj_on :
```
to match `card_image_eq_iff_inj_on`.

Reply:
`card_image_eq_iff_inj_on` is a bad name because there's only one RHS that makes sense in this context, so I would rather change it to match the new one.

Reply:
I don't feel too stronly - renaming the existing lemma is fine too, what matters to me is that they match.
Just make sure to mention it in the PR description.

Reply:
Renaming `card_image_eq_iff_inj_on` to `card_image_iff`.